### DISCUSSION TITLE
Add defer to <script> tag?

### DISCUSSION THREAD

**Discussion by bigskillet** [barrel/shopify-vite#138] [Q&A] [answered]
Is it possible to add `defer` to the `<script>` tag in the snippet?

**Comment by montalvomiguelo** [comment, marked as answer]
It would not be necessary; JavaScript modules `type="module"` are [deferred automatically](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts)

**Comment by RSeveryn** [comment]
@montalvomiguelo Is it possible to have the ability to add an 'async' or 'defer' attribute to the script tag with 'vite-tag' snippet?
